Cargo.toml

```diff
@@ -64,7 +64,7 @@ full = [
 ]
 
 [dependencies]
-teloxide-core = { version = "0.3.1", default-features = false }
+teloxide-core = { version = "0.3.3", default-features = false }
```

Comment:
`teloxide-core` has new features: `trace_adaptor`, `erased`. We should expose them in `teloxide` too.